packages/core/addon/templates/components/table-renderer-vertical-collection.hbs

```diff
@@ -99,10 +99,14 @@
                     column=column
                 }}
               {{else}}
+                {{#if (and (eq index 0) row.__meta__.isRollup)}}
+                  <div class="table-row-subtotal">Subtotal</div>
```

Comment:
Looking at the spec, this is 12px and the value is 14px. I think we are at 12px for the value :/ How would this look if the `Subtotal` title was a little smaller than the value font size.

Reply:
![Screen Shot 2021-08-09 at 12 56 50 PM](https://user-images.githubusercontent.com/99422/128751852-6059fdc4-cbd5-482f-9118-99d8719f03cf.png)
![Screen Shot 2021-08-09 at 12 57 08 PM](https://user-images.githubusercontent.com/99422/128751857-5ef36082-bddf-4587-9754-f4ca45ffa56e.png)

Here it is at 11 and 10. I think 11 looks better, 10 looks a bit too small for me.

Reply:
was decided 11px for now.